cmd/cofidectl/cmd/federation/federation.go

```diff
@@ -83,6 +101,52 @@ func (c *FederationCommand) GetListCommand() *cobra.Command {
 	return cmd
 }
 
+type bundles struct {
+	serverCABundle   string
+	federatedBundles map[string]string
+}
+
+// checkFederationStatus builds a comparison map between two trust domains, retrieves there server CA bundle and any federated bundles available
+// locally from the SPIRE server, and then compares the bundles on each to verify SPIRE has the correct bundles on each side of the federation
+func checkFederationStatus(ctx context.Context, kubeConfig string, from *trust_zone_proto.TrustZone, to *trust_zone_proto.TrustZone) string {
+	compare := make(map[*trust_zone_proto.TrustZone]bundles)
+
+	for _, tz := range []*trust_zone_proto.TrustZone{from, to} {
+		client, _ := kubeutil.NewKubeClientFromSpecifiedContext(kubeConfig, tz.GetKubernetesContext())
+		serverCABundle, federatedBundles, err := spire.GetServerCABundleAndFederatedBundles(ctx, client)
+		if err != nil {
+			return "Unknown"
```

Comment:
Should we distinguish between a failure at this point and an unexpected bundle setup? i.e. propagate this error.